app/controllers/api/v1/session.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+require 'barong/security/access_token'
+
+module API
+  module V1
+    class Session < Grape::API
+      desc 'Session related routes'
+      resource :session do
+        post '/create' do
+          acc = Account.find_by(email: params[:email])
+          return error!('401 Unauthorized', 401) unless acc
```

Comment:
Makes no sense always to respond with different 401 messages. People can use it to guess what emails or apps exist. Always respond with 401 and typical message (or better no message).